hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -317,6 +317,7 @@ public void updateRegionStats(ServerName serverName, byte[] regionName, RegionLo
   protected final Counter hedgedReadWin;
   protected final Histogram concurrentCallsPerServerHist;
   protected final Histogram numActionsPerServerHist;
+  protected final Timer backoffTimer; // TODO(baugenreich) should this be a timer??
```

Comment:
Im thinking timer is the appropriate metric here.

Reply:
Ya I think timer makes sense